src/mgmt/manager/project/api/nodes.py

```diff
@@ -179,16 +216,18 @@ def all_nodes():
             zgc = Zgc.query.filter_by(zgc_id=post_data['zgc_id']).first()
             if zgc is not None:
                 gws = getGWsFromIpRange(zgc.ip_start, zgc.ip_end)
-                create_droplet(name='droplet-' +'tenant-' + post_data['name'].replace('_', "-").lower(), 
+                create_droplet(name='droplet-tenant-' + post_data['name'].replace('_', "-").lower(), 
                                ip=[gw['ip'] for gw in gws], 
                                mac=[gw['mac'] for gw in gws],
-                               itf=post_data['inf_tenant'], 
+                               itf=post_data['inf_tenant'],
+                               node_ip= post_data['ip_control'],
```

Comment:
Nice, also had this in my PR 😊

Reply:
honestly, I didn't have time to see ur change yet, but it's very cool we have many common thoughts